### DISCUSSION TITLE
default values not working

### DISCUSSION THREAD

**Discussion by YaminMECHQI** [filamentphp/filament#4026] [General]
hello i'm using filament on my project and now the default value of toggle, radio, Repeater .. not working

**Comment by danharrin** [comment]
In admin panel resources?

**Reply by Yamin-MECHQI** [reply]
Yes

**Reply by danharrin** [reply]
I see you opened an issue, but you didnt provide a reproduction repo. Edit the issue and it will reopen. Then we can investigate the bug and fix it.

**Reply by YaminMECHQI** [reply]
https://github.com/filamentphp/filament/issues/4039
